### PR TITLE
Update exclusions.R

### PR DESCRIPTION
**Pull request by MarioGuCBMR**
I have added some more exclusion criteria!

### CODE REVIEW

File: R/exclusions.R [hunk 1]
```diff
@@ -64,7 +64,7 @@ exclude_from_title <- function(data) {
     exclude(title, "open( |-) (human)? ?(genom(ic|e)s?|genetics?)") |>
     # exclude(title, "covid(-19)?|SARS-CoV-2") |>
     # Things specific to datasets that are open
-    exclude(title, "open( |-)(datasets?|data sets?|data portal|data framework|data platform|data resource|data exploration)") |>
+    exclude(title, "open( |-)(datasets?|data sets?|data portal|data framework|data platform|data resource|data exploration)|system?|") |>
```
Comment (lwjohnst86) [inline R/exclusions.R:67]:
```suggestion
    exclude(title, "open( |-)(datasets?|data sets?|data portal|data framework|data platform|data resource|data exploration|system)") |>
```

File: R/exclusions.R [hunk 2]
```diff
@@ -13,7 +13,7 @@ exclude_from_title <- function(data) {
     dplyr::arrange(dplyr::desc(publication_date)) |>
     dplyr::distinct(title, .keep_all = TRUE) |>
     # Items related to surgery
-    exclude(title, "surgery|surgical|angle glaucoma|fracture|abdomen|abdominal|wound|vein|resection|urethroplasty|ectomy|inguinal|aortic|rhinoplasty|lumbar") |>
+    exclude(title, "surgery|surgical|angle glaucoma|fracture|abdomen|abdominal|wound|vein|resection|urethroplasty|ectomy|inguinal|aortic|rhinoplasty|lumbar|health|clinical|disease|patient|medical") |>
```
Comment (lwjohnst86) [inline R/exclusions.R:16]: I think on their own, these exclusions might drop a lot more than we expect. I'll look into it though

File: R/exclusions.R [hunk 3]
```diff
@@ -38,7 +38,7 @@ exclude_from_title <- function(data) {
     # Items related to dissemination
     exclude(title, "open( |-|, )(access|review(er)?)|publishing|communication") |>
     # Items related to education
-    exclude(title, "open( |-|, )(textbooks?|doors?|education(al)?|seminar|learning|schools|course|book|pedagogy)") |>
+    exclude(title, "open( |-|, )(textbooks?|doors?|education(al)?|seminar|learning|schools|course|book|pedagogy|teaching|learning|online?)") |>
```
Comment (lwjohnst86) [inline R/exclusions.R:41]:
```suggestion
    exclude(title, "open( |-|, )(textbooks?|doors?|education(al)?|seminar|learning|schools|course|book|pedagogy|teaching|learning|online)") |>
```

**Review by lwjohnst86 — CHANGES_REQUESTED** [review]
Some comments and suggested changes.

File: R/exclusions.R (hunk 3 above)
Comment (MarioGuCBMR) [reply R/exclusions.R:41]: I think those are great!! I just asked chatGPT to get me from a sample of 200 titles, the 5 main topics across the titles and, from those, the top 5 key words. A whole section was about education and all the changes here were included. Let's do this!!

File: R/exclusions.R (hunk 2 above)
Comment (MarioGuCBMR) [reply R/exclusions.R:16]: Definitely!!

**Pull request closed without merging**